src/transaction.cpp

```diff
@@ -214,12 +214,18 @@ namespace mamba
         }
         m_transaction = solver_create_transaction(solver);
         init();
+        Console::instance().json_down("actions");
     }
 
     MTransaction::~MTransaction()
     {
         LOG(INFO) << "Freeing transaction.";
         transaction_free(m_transaction);
+        Console::instance().json_write("PREFIX", Context::instance().target_prefix);
```

Comment:
Can we simplify this to use the fancy nlohmann::json syntax (

```
{
{"PREFIX", target_prefix},
{"dry_run", ...}
}
```

(This is the example in the readme:

```
// instead, you could also write (which looks very similar to the JSON above)
json j2 = {
  {"pi", 3.141},
  {"happy", true},
  {"name", "Niels"},
  {"nothing", nullptr},
  {"answer", {
    {"everything", 42}
  }},
  {"list", {1, 0, 2}},
  {"object", {
    {"currency", "USD"},
    {"value", 42.99}
  }}
};
```

Reply:
also on win, PREFIX is lowercase? should it be lowercase or uppercase?

Reply:
ah, wait, that isn't true, just saw the lowercase prefix below, sorry :)

Reply:
Regarding the more concise syntax, it's possible.